modules/network-observability-loki-install.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-loki-installation_{context}"]
+
+.Prerequisites
+
+* Installed Loki Operator version 5.6, which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
+* Object storage, such as AWS S3
+* {product-title} 4.10+
+* Linux Kernel 4.18
+
+//* <Any Loki install prerequisites for using with Network Observability operator?>
+
+== Installing the Loki Operator 
```

Comment:
@jotak @mariomac @jpinsonneau what if Loki Operator is already installed on the cluster? does it need to be modified to work with NetObserv?

Reply:
Hmmm, but if it is installed first, it won't be in `netobserv` namespace, which we recommend below - is this not needed?

Reply:
I think it is _just_ a recommendation, not a necessity. If you don't have it in the same namespace I think the main implication is that if you're following all our setup guidance, you'll have to make sure you specify your own namespace in all the areas where we've specified netobserv.